From 1969b3c60db675040ec0d1b09698807647aac7ed Mon Sep 17 00:00:00 2001
Message-Id: <1969b3c60db675040ec0d1b09698807647aac7ed.1649661930.git.geliang.tang@suse.com>
From: Florent Revest <revest@chromium.org>
Date: Wed, 14 Apr 2021 17:56:32 +0200
Subject: [PATCH] selftests/bpf: Fix the ASSERT_ERR_PTR macro
Git-commit: 1969b3c60db675040ec0d1b09698807647aac7ed
Patch-mainline: v5.13-rc1
References: bsn#127
Modified-by-SEL: No


It is just missing a ';'. This macro is not used by any test yet.

Fixes: 22ba36351631 ("selftests/bpf: Move and extend ASSERT_xxx() testing macros")
Signed-off-by: Florent Revest <revest@chromium.org>
Signed-off-by: Andrii Nakryiko <andrii@kernel.org>
Acked-by: Martin KaFai Lau <kafai@fb.com>
Link: https://lore.kernel.org/bpf/20210414155632.737866-1-revest@chromium.org
Signed-off-by: Geliang Tang <geliang.tang@suse.com>
---
 tools/testing/selftests/bpf/test_progs.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/testing/selftests/bpf/test_progs.h b/tools/testing/selftests/bpf/test_progs.h
index e87c8546230e..ee7e3b45182a 100644
--- a/tools/testing/selftests/bpf/test_progs.h
+++ b/tools/testing/selftests/bpf/test_progs.h
@@ -210,7 +210,7 @@ extern int test__join_cgroup(const char *path);
 #define ASSERT_ERR_PTR(ptr, name) ({					\
 	static int duration = 0;					\
 	const void *___res = (ptr);					\
-	bool ___ok = IS_ERR(___res)					\
+	bool ___ok = IS_ERR(___res);					\
 	CHECK(!___ok, (name), "unexpected pointer: %p\n", ___res);	\
 	___ok;								\
 })
-- 
2.34.1

